### PR TITLE
CH:282 - Cluster capacity and storage component

### PR DESCRIPTION
**Pull request by sujit-h-singh**
# Problem Statement
- Updated design for cluster capacity and storage component.

# Design
-  [Figma-design](https://www.figma.com/file/OItRwHidQ2nci633ifHunH/Latest-Dashboard-Design?node-id=731%3A8446)

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [x] Interface change (if any) are documented
- [x] Side effects on other features (deployment/upgrade)
- [x] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [x] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by sujit-h-singh** [comment]
Test cases added for the capacity card and storage component card

File: manager/gui/src/components/dashboard/LrDashboardCapacityCard.vue [hunk 1]
```diff
@@ -66,11 +77,11 @@ export default class LrDashboardCapacityCard extends Vue {
   public capacityDetails: CapacityData = {} as CapacityData;
 
   public async mounted() {
-    const data = await Api.getData("/dashboard/capacity", {
+    const data: any = await Api.getData("/dashboard/capacity", {
```
Comment (NareshGopal) [inline manager/gui/src/components/dashboard/LrDashboardCapacityCard.vue:80]: 'data' can be renamed as 'res' or 'response'. Coz in the next line, it is called "data.data".

**Review by NareshGopal — CHANGES_REQUESTED** [review]
1. Latest icons are not updated in the performance widget
2. Alignment of the widgets is not proper
3. Alignment of the alert type list is not proper in 720p
4. Latest icon is not updated in the Background activities widget

File: manager/gui/src/components/dashboard/LrDashboardCapacityCard.vue (hunk 1 above)
Comment (sujit-h-singh) [reply manager/gui/src/components/dashboard/LrDashboardCapacityCard.vue]: Changed

**Review by NareshGopal — APPROVED** [review]

**Pull request merged**